README.md

```diff
@@ -1,167 +1,169 @@
+
+### Installation and updates
+* Install this application by downloading and running the latest [release from here](https://github.com/BigSpice/VTOL/releases)
+   * ``Download the .msi package``
 
-Install this application by downloading and running the latest release:
-<br /><br />
 ![image](https://user-images.githubusercontent.com/23240514/150025911-aacaf334-cc33-4239-94d7-512cfb84fe49.png)
-<br /><br />
-If you already use this application, consider navigating to "check for updates" inside the application to download and install the latest release.<br /><br />
-![image](https://user-images.githubusercontent.com/23240514/152447226-88662295-b7a9-4fd5-a958-dfc5ce59bd9c.png)<br /><br />
-Once the application successful downloaded and installed:
-<br /><br />
-<br /><br /><br />
-
-1. Configure your Titanfall 2 installation directory.
-   <br /><br />
-   If the automatic browse cannot find your Titanfall 2 installation, please select "browse" and _browse_ to where your Titanfall 2 installation is, and select it.
-   <br /><br />
-   ![image](https://user-images.githubusercontent.com/23240514/152446386-4bb584c1-038a-4dd6-9588-ac3827884144.png)<br /><br />
-2. Verify, or install the [Northstar Client](https://northstar.tf)
-   <br /><br />
-   ![image](https://user-images.githubusercontent.com/23240514/152446412-d10f1f7e-bbc2-4558-8e8b-033aae2af410.png)<br /><br />
-   Use this to verify your existing Northstar installation.
-   <br /><br />
-   _By using that, the application will download the latest [Northstar release](https://github.com/R2Northstar/Northstar/releases/latest)._
-   <br /><br />
-   ![image](https://user-images.githubusercontent.com/23240514/152446466-26d91f33-948d-4051-9247-c21b20328515.png)
 
-<br /><br />
+* This application has an auto-update Feature in the Update section
+
+![image](https://user-images.githubusercontent.com/23240514/152447226-88662295-b7a9-4fd5-a958-dfc5ce59bd9c.png)
+
+### Configuration
+
+ * If the launcher fails to automatically find your Titanfall 2 installation, please select ``Browse`` on the Main page and _browse_ to where your Titanfall 2 installation is, and select the Titnafall2 folder
+
+ * Common Game Install Locations:
+   *    Steam: `C:\Program Files (x86)\Steam\steamapps\common\Titanfall2`
+   *    Origin: `C:\Program Files (x86)\Origin Games\Titanfall2` 
+
 
-_Installations will automatically be updated if an outdated version is detected, repairing will be done if the version is the same._
-<br /><br /><br /><br />
+   ![image](https://user-images.githubusercontent.com/23240514/152446386-4bb584c1-038a-4dd6-9588-ac3827884144.png)
 
-<h1> MOD BROWSER </h1>
 
-_The [Thunderstore](https://northstar.thunderstore.io) integration has been released in v1.1.7_<br />
-<br />
+### Verify [Northstar](https://northstar.tf)
+
+* Point the ``+Northstar install Exe Directory`` field to where northstar is installed: 
```

Comment:
preferably change this to `* Select the NorthstarLauncher.exe file in the Northstar installation field:`